resources/views/hardware/edit.blade.php

```diff
@@ -290,43 +290,213 @@ function user_add(status_id) {
     $(document).ready(function() {
 
         var max_fields      = 100; //maximum input boxes allowed
+		var max_value		= 100000000000; //maximum allowed value
```

Comment:
How are you determining that number?

Reply:
It is maximum number where value is properly managed. Over it generated number can be incorrect.